packages/contracts-ts/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "@eth-optimism/contracts-ts",
```

Comment:
Better name?  what about `@eth-optimism/ts`?

Reply:
what if we call it `op-bindings-ts`?

Reply:
Hmmm. I like this the best internally don't think it's a good external name though not having Optimism in the namespace

Reply:
I think `@eth-optimism/contracts-ts` is pretty clear and not that bad of a name tbh

Reply:
yeah contracts-ts is boring and clear enough